state/indexer/sink/psql/schema.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TYPE block_event_type AS ENUM ('begin_block', 'end_block', '');
```

Comment:
Should these tables have association to chain-id? That way when the chain is upgraded a new table would be used to clearly separate the data.

Reply:
A new table? If we can easily get access to that data in the indexer, we should add it.

Reply:
updated event tables with chain-id.

Reply:
updated event tables with a chain-id column.